src/FffPolygonGenerator.cpp

```diff
@@ -1029,21 +1029,28 @@ void FffPolygonGenerator::processPlatformAdhesion(SliceDataStorage& storage)
 
     Polygons first_layer_outline;
     coord_t primary_line_count;
-    //If brim for prime tower is used, add the brim for prime tower separately.
-    //Since FffGCodeWriter assumes that the outermost contour is last, add this first. Keep it ordered!
-    bool should_brim_prime_tower = storage.primeTower.enabled && mesh_group_settings.get<bool>("prime_tower_brim_enable");
-    if (should_brim_prime_tower)
+
+    EPlatformAdhesion adhesion_type = mesh_group_settings.get<EPlatformAdhesion>("adhesion_type");
+
+    // The switch statement is not used here, since if brim for the prime tower is needed, it should go after any skirt, but before any brim.
+    if (adhesion_type == EPlatformAdhesion::SKIRT)
+    {
```

Comment:
```suggestion
    if (adhesion_type == EPlatformAdhesion::SKIRT)
    {
        // If the prime tower needs a brim, tower brim should go after any skirt, but before any model brim.
```

Had to read this comment 5 times. I hope my suggestion is more clear :-)

Reply:
Feel free to change this!